### DISCUSSION TITLE
Hayward 24VAC Transformer Power Rating

### DISCUSSION THREAD

**Discussion by celestinjr** [tagyoureit/nodejs-poolController#950] [Q&A]
Does anyone know the power rating of the Hayward 24VAC transformer?  The one in my Aqualogic AQL-PS-8 has a "085013H-2 T" label on it.  I've been unable to find any specifications online, but it seems the same transformer is used across the Aquarite and 4-16 relay Aqualogic panels under the part number "GLX-XFMR".  I measured 10ish amps on the 24VAC line when powering the salt cell only.

Thanks!

**Comment by mhmatttu** [comment]
I will be cannibalizing a failed Hayward OCP and install nixie following @CodeSlinger69 work so was researching this transformer as well. I found this linked discussion on TFP and inquired about the fuse and varsistors. 

https://www.troublefreepool.com/threads/aqua-logic-transformer.288026/

There is no mention of fuses or varsistors in relation to the recommended [24vac transformer](https://www.amazon.com/Functional-Devices-TR75VA005-Transformer-Threaded/dp/B007IAVI74/ref=sr_1_48?dchild=1&gclid=EAIaIQobChMI1cmP7IWd8gIV0wN9Ch1gPwamEAMYASAAEgLz6vD_BwE&hvadid=410001655388&hvdev=c&hvlocphy=9032516&hvnetw=g&hvqmt=e&hvrand=13109899113586692443&hvtargid=kwd-2480373819&hydadcr=24658_11410762&keywords=24vac+transformer&qid=1628275244&sr=8-48) in the Nixie wiki so curious if they may be integrated elsewhere or truly not needed.
